petastorm/spark/spark_dataset_converter.py

```diff
@@ -112,6 +116,39 @@ def _delete_cache_data_atexit(dataset_url):
         warnings.warn('delete cache data {url} failed.'.format(url=dataset_url))
 
 
+def get_env_rank_and_size():
```

Comment:
Let's prefix with `_` as both `get_env_rank_and_size` and `is_rank_and_size_consistent_with_env` are private.